### ISSUE TITLE
Sell to_open=False results in short sell

### ISSUE THREAD

**Issue by fny** [alienbrett/PyAlly#7]
I just tested out a bogus sale with the following to see if everything worked.

```
    order = ally.order.Order(
        type = ally.order.Sell(to_open=False),
        timespan = ally.order.Timespan('day'),
        price = ally.order.Limit(price),
        instrument = ally.instrument.Equity(ticker),
        quantity = ally.order.Quantity(qty)
    )
    a.submit_order(order, preview=False)
```

I did expect it to bounce back... but it looks like this is being interpreted as a short sell rather than closing a position.

```
Sell Short | 100 | ABC | ALPHABETSOUP INC | Limit $187.14 | 194.92 | 194.95 | Day |   | 01/31/20 3:40 PM ET | SVR-6091077263 | RejectedThe limit price you have entered is significantly away from the current market price.
```

ally.order.Sell(to_open=True) makes it a sell to close.

Also, that API is dangerously confusing.

**Comment by eyesuk** [comment]
Any update on this? Can we send you beer money to help

**Comment by alienbrett** [comment]
Spotted, thanks for the bug report

**Comment by alienbrett** [comment]
Fixed in the latest V0.3.0
